.github/workflows/tests.yml

```diff
@@ -0,0 +1,130 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run tests / create artifacts
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, ready_for_review]
+    branches: [ master ]
+
+jobs:
+  test-build-deploy:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./cdk
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Setup nodejs
+      uses: actions/setup-node@v2
+      with:
+        node-version: 16
+    - name: Install aws-cdk
+      run: npm install -g aws-cdk@1.110.1
```

Comment:
Probably a follow-up, but I wonder if we can simplify the version constraint sharing with setup.py, etc.